### PR TITLE
Updates webpack and micromatch

### PR DESCRIPTION
**Pull request by SaptakS**
## Description

Updates webpack. https://github.com/advisories/GHSA-4vvj-4cpr-p986
Updates micromatch: https://github.com/advisories/GHSA-952p-6rrq-rcjv

## Type of change

- [ ] Bug fix
- [ ] New feature
- [x] Vulnerabilities update
- [ ] Config changes
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires an admin update after deploy
- [ ] Includes a database migration removing  or renaming a field

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -59,7 +59,7 @@ jobs:
     environment:
       SETUPTOOLS_USE_DISTUTILS: stdlib
     machine:
-      image: ubuntu-2204:2023.07.2
```
Comment (harrislapiroff) [inline .circleci/config.yml:62]: Is this intentionally part of this commit? (Not sure if it makes a difference one way or another, but curious about it)

**Review by harrislapiroff — APPROVED** [review]
Curious about the motivation for the ubuntu upgrade, but it seems like a good idea! All LGTM

File: .circleci/config.yml (hunk 1 above)
Comment (SaptakS) [reply .circleci/config.yml:62]: Yes! Circle CI has deprecated some images and these are one of them. So the CI was failing without updating the images.

Comment (SaptakS) [reply .circleci/config.yml:62]: https://discuss.circleci.com/t/linux-image-deprecations-and-eol-for-2024/50177

Comment (harrislapiroff) [reply .circleci/config.yml:62]: Ah, makes sense!

**Pull request merged**